cnv/cnv_users_guide/cnv-openshift-cluster-monitoring.adoc

```diff
@@ -0,0 +1,18 @@
+[id="cnv-openshift-cluster-monitoring"]
+= {product-title} cluster monitoring
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-openshift-cluster-monitoring
+toc::[]
+
+{product-title} provides various resources for monitoring at the cluster level.
+
+See the {product-title} documentation for more information:
```

Comment:
I still think we need a sentence or two about why the OCP bits are enough and/or why each of these OCP functions is good/useful/important for CNV. I'd add a sentence or two of context for each of these links, if you have the info.

Reply:
@kalexand-rh - sorry for the delay. I've "borrowed" some text from the modules/assemblies of the relevant links to provide that context. 
If you're happy with that, then I think this PR is ready. Thank you!